<Xcpt>
  <Time>27.04.2017 08:23:23</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 08:25:09</Time>
  <Exception>
    <Source>CardGame.DAL</Source>
    <Message>User Does not Exist</Message>
    <Stack>   bei CardGame.DAL.Logic.UserManager.GetUserByUserEmail(String email) in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 124.</Stack>
  </Exception>
</Xcpt>
<Info>
  <Time>27.04.2017 08:25:39</Time>
  <Info>Entered Pass = Christian</Info>
</Info>
<Info>
  <Time>27.04.2017 08:25:39</Time>
  <Info>HashPass = 0D6F1304B6C817ADC9E9BC5B0036E9FBD627DE7998F69A9865BC9FA9530C17295C8FF91406A620FC3E7C8D8F331E8FFB0D6754E06750738718AEDF82ADF05292</Info>
</Info>
<Info>
  <Time>27.04.2017 08:25:50</Time>
  <Info>Entered Pass = Christian</Info>
</Info>
<Info>
  <Time>27.04.2017 08:25:50</Time>
  <Info>HashPass = 0D6F1304B6C817ADC9E9BC5B0036E9FBD627DE7998F69A9865BC9FA9530C17295C8FF91406A620FC3E7C8D8F331E8FFB0D6754E06750738718AEDF82ADF05292</Info>
</Info>
<Info>
  <Time>27.04.2017 08:26:17</Time>
  <Info>Entered Pass = christian</Info>
</Info>
<Info>
  <Time>27.04.2017 08:26:17</Time>
  <Info>HashPass = 9EA2CB560933A21219C6BA2A8E0349BF7F2E37214E8C1A8C598B003EE8244B2358F33FC1E2CA633459FEC20E8E9BDF92B83A08450C75421D995D8B20C25FD123</Info>
</Info>
<Info>
  <Time>27.04.2017 08:27:11</Time>
  <Info>Entered Pass = Christian</Info>
</Info>
<Info>
  <Time>27.04.2017 08:27:11</Time>
  <Info>HashPass = 0D6F1304B6C817ADC9E9BC5B0036E9FBD627DE7998F69A9865BC9FA9530C17295C8FF91406A620FC3E7C8D8F331E8FFB0D6754E06750738718AEDF82ADF05292</Info>
</Info>
<Info>
  <Time>27.04.2017 08:27:29</Time>
  <Info>Entered Pass = Christian</Info>
</Info>
<Info>
  <Time>27.04.2017 08:27:29</Time>
  <Info>HashPass = 0D6F1304B6C817ADC9E9BC5B0036E9FBD627DE7998F69A9865BC9FA9530C17295C8FF91406A620FC3E7C8D8F331E8FFB0D6754E06750738718AEDF82ADF05292</Info>
</Info>
<Info>
  <Time>27.04.2017 08:27:41</Time>
  <Info>Entered Pass = Christian</Info>
</Info>
<Info>
  <Time>27.04.2017 08:27:41</Time>
  <Info>HashPass = 0D6F1304B6C817ADC9E9BC5B0036E9FBD627DE7998F69A9865BC9FA9530C17295C8FF91406A620FC3E7C8D8F331E8FFB0D6754E06750738718AEDF82ADF05292</Info>
</Info>
<Info>
  <Time>27.04.2017 08:27:58</Time>
  <Info>Entered Pass = christian</Info>
</Info>
<Info>
  <Time>27.04.2017 08:27:58</Time>
  <Info>HashPass = 9EA2CB560933A21219C6BA2A8E0349BF7F2E37214E8C1A8C598B003EE8244B2358F33FC1E2CA633459FEC20E8E9BDF92B83A08450C75421D995D8B20C25FD123</Info>
</Info>
<Xcpt>
  <Time>27.04.2017 08:27:58</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 09:34:18</Time>
  <Exception>
    <Source>CardGame.DAL</Source>
    <Message>User gibt es noch nicht, bitte Registrieren</Message>
    <Stack>   bei CardGame.DAL.Logic.AuthManager.AuthUser(String email, String password) in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\AuthManager.cs:Zeile 80.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 09:34:31</Time>
  <Exception>
    <Source>CardGame.DAL</Source>
    <Message>User gibt es noch nicht, bitte Registrieren</Message>
    <Stack>   bei CardGame.DAL.Logic.AuthManager.AuthUser(String email, String password) in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\AuthManager.cs:Zeile 80.</Stack>
  </Exception>
</Xcpt>
<Info>
  <Time>27.04.2017 09:34:52</Time>
  <Info>Entered Pass = Thomas</Info>
</Info>
<Info>
  <Time>27.04.2017 09:34:52</Time>
  <Info>HashPass = 796B941E70BE495B935234AAA12B183B121E24EE6B1BBB8AFC7F8B94D3DC87D62DCA9FB55B122E15CAE00E22E4625AC6A7B8E3FBF8EBF2E9F9F2D55E6F0E322F</Info>
</Info>
<Info>
  <Time>27.04.2017 09:35:00</Time>
  <Info>Entered Pass = thomas</Info>
</Info>
<Info>
  <Time>27.04.2017 09:35:00</Time>
  <Info>HashPass = 4FE2555B70C8288A1B40B53BF389D863C1421E7CFF58160CB57B2644E323675418B628F32B33B587BEFC2970E72698C65739899AF849B41642D017423A2B7461</Info>
</Info>
<Info>
  <Time>27.04.2017 09:35:53</Time>
  <Info>Entered Pass = Thomas</Info>
</Info>
<Info>
  <Time>27.04.2017 09:35:53</Time>
  <Info>HashPass = 796B941E70BE495B935234AAA12B183B121E24EE6B1BBB8AFC7F8B94D3DC87D62DCA9FB55B122E15CAE00E22E4625AC6A7B8E3FBF8EBF2E9F9F2D55E6F0E322F</Info>
</Info>
<Info>
  <Time>27.04.2017 09:36:08</Time>
  <Info>Entered Pass = Thomas</Info>
</Info>
<Info>
  <Time>27.04.2017 09:36:08</Time>
  <Info>HashPass = 796B941E70BE495B935234AAA12B183B121E24EE6B1BBB8AFC7F8B94D3DC87D62DCA9FB55B122E15CAE00E22E4625AC6A7B8E3FBF8EBF2E9F9F2D55E6F0E322F</Info>
</Info>
<Info>
  <Time>27.04.2017 09:36:14</Time>
  <Info>Entered Pass = thomas</Info>
</Info>
<Info>
  <Time>27.04.2017 09:36:14</Time>
  <Info>HashPass = 4FE2555B70C8288A1B40B53BF389D863C1421E7CFF58160CB57B2644E323675418B628F32B33B587BEFC2970E72698C65739899AF849B41642D017423A2B7461</Info>
</Info>
<Xcpt>
  <Time>27.04.2017 09:36:39</Time>
  <Exception>
    <Source>CardGame.DAL</Source>
    <Message>User gibt es noch nicht, bitte Registrieren</Message>
    <Stack>   bei CardGame.DAL.Logic.AuthManager.AuthUser(String email, String password) in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\AuthManager.cs:Zeile 80.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 09:37:38</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Info>
  <Time>27.04.2017 09:40:06</Time>
  <Info>GetNumUsers 27</Info>
</Info>
<Info>
  <Time>27.04.2017 09:40:07</Time>
  <Info>GetNumCards 382</Info>
</Info>
<Info>
  <Time>27.04.2017 09:40:07</Time>
  <Info>GetNumDecks 25</Info>
</Info>
<Xcpt>
  <Time>27.04.2017 10:00:19</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 10:27:59</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 11:23:20</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Info>
  <Time>27.04.2017 12:38:11</Time>
  <Info>Entered Pass = Christian</Info>
</Info>
<Info>
  <Time>27.04.2017 12:38:11</Time>
  <Info>HashPass = 0D6F1304B6C817ADC9E9BC5B0036E9FBD627DE7998F69A9865BC9FA9530C17295C8FF91406A620FC3E7C8D8F331E8FFB0D6754E06750738718AEDF82ADF05292</Info>
</Info>
<Info>
  <Time>27.04.2017 12:38:20</Time>
  <Info>Entered Pass = Christian</Info>
</Info>
<Info>
  <Time>27.04.2017 12:38:20</Time>
  <Info>HashPass = 0D6F1304B6C817ADC9E9BC5B0036E9FBD627DE7998F69A9865BC9FA9530C17295C8FF91406A620FC3E7C8D8F331E8FFB0D6754E06750738718AEDF82ADF05292</Info>
</Info>
<Info>
  <Time>27.04.2017 12:38:36</Time>
  <Info>Entered Pass = Christian</Info>
</Info>
<Info>
  <Time>27.04.2017 12:38:36</Time>
  <Info>HashPass = 0D6F1304B6C817ADC9E9BC5B0036E9FBD627DE7998F69A9865BC9FA9530C17295C8FF91406A620FC3E7C8D8F331E8FFB0D6754E06750738718AEDF82ADF05292</Info>
</Info>
<Info>
  <Time>27.04.2017 12:38:47</Time>
  <Info>Entered Pass = christian</Info>
</Info>
<Info>
  <Time>27.04.2017 12:38:47</Time>
  <Info>HashPass = 9EA2CB560933A21219C6BA2A8E0349BF7F2E37214E8C1A8C598B003EE8244B2358F33FC1E2CA633459FEC20E8E9BDF92B83A08450C75421D995D8B20C25FD123</Info>
</Info>
<Xcpt>
  <Time>27.04.2017 12:38:47</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 12:48:32</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 13:03:20</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 13:04:40</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 13:08:09</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 13:11:49</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Info>
  <Time>27.04.2017 13:13:26</Time>
  <Info>id: 8</Info>
</Info>
<Info>
  <Time>27.04.2017 13:13:26</Time>
  <Info>numPacks: 1</Info>
</Info>
<Info>
  <Time>27.04.2017 13:13:27</Time>
  <Info>vID: 382</Info>
</Info>
<Info>
  <Time>27.04.2017 13:13:27</Time>
  <Info>Card: 495</Info>
</Info>
<Info>
  <Time>27.04.2017 13:13:27</Time>
  <Info>Card: 459</Info>
</Info>
<Info>
  <Time>27.04.2017 13:13:27</Time>
  <Info>Card: 706</Info>
</Info>
<Info>
  <Time>27.04.2017 13:13:27</Time>
  <Info>Card: 622</Info>
</Info>
<Info>
  <Time>27.04.2017 13:13:27</Time>
  <Info>Card: 495</Info>
</Info>
<Info>
  <Time>27.04.2017 13:13:51</Time>
  <Info>vID: 382</Info>
</Info>
<Info>
  <Time>27.04.2017 13:13:51</Time>
  <Info>Card: 681</Info>
</Info>
<Info>
  <Time>27.04.2017 13:13:51</Time>
  <Info>Card: 690</Info>
</Info>
<Info>
  <Time>27.04.2017 13:13:51</Time>
  <Info>Card: 318</Info>
</Info>
<Info>
  <Time>27.04.2017 13:13:51</Time>
  <Info>Card: 239</Info>
</Info>
<Info>
  <Time>27.04.2017 13:13:51</Time>
  <Info>Card: 540</Info>
</Info>
<Xcpt>
  <Time>27.04.2017 13:14:24</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 13:28:31</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 13:31:00</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 13:33:14</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 13:43:18</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 13:47:47</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 14:40:59</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 14:44:52</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 14:54:40</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 14:57:31</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 15:00:43</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 15:01:57</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 15:07:08</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 15:38:03</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 15:41:11</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 15:47:13</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 15:49:45</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 15:54:42</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 16:02:56</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 16:06:07</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\faulthom\Documents\Github\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 18:03:31</Time>
  <Exception>
    <Source>CardGame.DAL</Source>
    <Message>User gibt es noch nicht, bitte Registrieren</Message>
    <Stack>   bei CardGame.DAL.Logic.AuthManager.AuthUser(String email, String password) in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\AuthManager.cs:Zeile 80.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 18:03:41</Time>
  <Exception>
    <Source>CardGame.DAL</Source>
    <Message>User gibt es noch nicht, bitte Registrieren</Message>
    <Stack>   bei CardGame.DAL.Logic.AuthManager.AuthUser(String email, String password) in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\AuthManager.cs:Zeile 80.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 18:04:29</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Info>
  <Time>27.04.2017 18:05:38</Time>
  <Info>Entered Pass = stefan</Info>
</Info>
<Info>
  <Time>27.04.2017 18:05:38</Time>
  <Info>HashPass = D26F2BAD2D39019187121BE939D72729130D070AEFF8647926D9720F12AAA433C1B9B7783C462DDBC97EB8D198095021761C5E8242D434BBA2C146B51FE57BCC</Info>
</Info>
<Xcpt>
  <Time>27.04.2017 18:09:34</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Info>
  <Time>27.04.2017 18:09:51</Time>
  <Info>id: 8</Info>
</Info>
<Info>
  <Time>27.04.2017 18:09:51</Time>
  <Info>numPacks: 1</Info>
</Info>
<Info>
  <Time>27.04.2017 18:09:52</Time>
  <Info>vID: 382</Info>
</Info>
<Info>
  <Time>27.04.2017 18:09:52</Time>
  <Info>Card: 471</Info>
</Info>
<Info>
  <Time>27.04.2017 18:09:52</Time>
  <Info>Card: 257</Info>
</Info>
<Info>
  <Time>27.04.2017 18:09:52</Time>
  <Info>Card: 422</Info>
</Info>
<Info>
  <Time>27.04.2017 18:09:52</Time>
  <Info>Card: 96</Info>
</Info>
<Info>
  <Time>27.04.2017 18:09:52</Time>
  <Info>Card: 591</Info>
</Info>
<Xcpt>
  <Time>27.04.2017 18:09:52</Time>
  <Exception>
    <Source>System</Source>
    <Message>Fehler beim Senden von Mail.</Message>
    <Stack>   bei System.Net.Mail.SmtpClient.Send(MailMessage message)
   bei CardGame.Web.HtmlHelpers.EmailHelper.SendEmail(String emailaddress, String emailsubject, String text) in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.Web\HtmlHelpers\EmailHelper.cs:Zeile 38.</Stack>
    <InnerException>
      <Source>System</Source>
      <Message>Der Remotename konnte nicht aufgelöst werden: 'srv08.itccn.loc'</Message>
      <Stack>   bei System.Net.ServicePoint.GetConnection(PooledStream PooledStream, Object owner, Boolean async, IPAddress&amp; address, Socket&amp; abortSocket, Socket&amp; abortSocket6)
   bei System.Net.PooledStream.Activate(Object owningObject, Boolean async, GeneralAsyncDelegate asyncCallback)
   bei System.Net.ConnectionPool.GetConnection(Object owningObject, GeneralAsyncDelegate asyncCallback, Int32 creationTimeout)
   bei System.Net.Mail.SmtpConnection.GetConnection(ServicePoint servicePoint)
   bei System.Net.Mail.SmtpClient.GetConnection()
   bei System.Net.Mail.SmtpClient.Send(MailMessage message)</Stack>
    </InnerException>
  </Exception>
</Xcpt>
<Info>
  <Time>27.04.2017 18:10:02</Time>
  <Info>id: 12</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:02</Time>
  <Info>numPacks: 1</Info>
</Info>
<Xcpt>
  <Time>27.04.2017 18:10:03</Time>
  <Exception>
    <Source>System</Source>
    <Message>Fehler beim Senden von Mail.</Message>
    <Stack>   bei System.Net.Mail.SmtpClient.Send(MailMessage message)
   bei CardGame.Web.HtmlHelpers.EmailHelper.SendEmail(String emailaddress, String emailsubject, String text) in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.Web\HtmlHelpers\EmailHelper.cs:Zeile 38.</Stack>
    <InnerException>
      <Source>System</Source>
      <Message>Der Remotename konnte nicht aufgelöst werden: 'srv08.itccn.loc'</Message>
      <Stack>   bei System.Net.ServicePoint.GetConnection(PooledStream PooledStream, Object owner, Boolean async, IPAddress&amp; address, Socket&amp; abortSocket, Socket&amp; abortSocket6)
   bei System.Net.PooledStream.Activate(Object owningObject, Boolean async, GeneralAsyncDelegate asyncCallback)
   bei System.Net.ConnectionPool.GetConnection(Object owningObject, GeneralAsyncDelegate asyncCallback, Int32 creationTimeout)
   bei System.Net.Mail.SmtpConnection.GetConnection(ServicePoint servicePoint)
   bei System.Net.Mail.SmtpClient.GetConnection()
   bei System.Net.Mail.SmtpClient.Send(MailMessage message)</Stack>
    </InnerException>
  </Exception>
</Xcpt>
<Info>
  <Time>27.04.2017 18:10:10</Time>
  <Info>id: 9</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:10</Time>
  <Info>numPacks: 1</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:11</Time>
  <Info>vID: 382</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:11</Time>
  <Info>Card: 243</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:11</Time>
  <Info>Card: 23</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:11</Time>
  <Info>Card: 207</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:11</Time>
  <Info>Card: 591</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:11</Time>
  <Info>Card: 114</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:11</Time>
  <Info>Card: 697</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:11</Time>
  <Info>Card: 92</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:11</Time>
  <Info>Card: 259</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:11</Time>
  <Info>Card: 112</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:11</Time>
  <Info>Card: 5</Info>
</Info>
<Xcpt>
  <Time>27.04.2017 18:10:11</Time>
  <Exception>
    <Source>System</Source>
    <Message>Fehler beim Senden von Mail.</Message>
    <Stack>   bei System.Net.Mail.SmtpClient.Send(MailMessage message)
   bei CardGame.Web.HtmlHelpers.EmailHelper.SendEmail(String emailaddress, String emailsubject, String text) in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.Web\HtmlHelpers\EmailHelper.cs:Zeile 38.</Stack>
    <InnerException>
      <Source>System</Source>
      <Message>Der Remotename konnte nicht aufgelöst werden: 'srv08.itccn.loc'</Message>
      <Stack>   bei System.Net.ServicePoint.GetConnection(PooledStream PooledStream, Object owner, Boolean async, IPAddress&amp; address, Socket&amp; abortSocket, Socket&amp; abortSocket6)
   bei System.Net.PooledStream.Activate(Object owningObject, Boolean async, GeneralAsyncDelegate asyncCallback)
   bei System.Net.ConnectionPool.GetConnection(Object owningObject, GeneralAsyncDelegate asyncCallback, Int32 creationTimeout)
   bei System.Net.Mail.SmtpConnection.GetConnection(ServicePoint servicePoint)
   bei System.Net.Mail.SmtpClient.GetConnection()
   bei System.Net.Mail.SmtpClient.Send(MailMessage message)</Stack>
    </InnerException>
  </Exception>
</Xcpt>
<Info>
  <Time>27.04.2017 18:10:33</Time>
  <Info>id: 9</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:33</Time>
  <Info>numPacks: 1</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:34</Time>
  <Info>vID: 382</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:34</Time>
  <Info>Card: 217</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:34</Time>
  <Info>Card: 625</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:34</Time>
  <Info>Card: 253</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:34</Time>
  <Info>Card: 657</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:34</Time>
  <Info>Card: 235</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:34</Time>
  <Info>Card: 270</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:34</Time>
  <Info>Card: 617</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:34</Time>
  <Info>Card: 610</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:34</Time>
  <Info>Card: 591</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:34</Time>
  <Info>Card: 495</Info>
</Info>
<Xcpt>
  <Time>27.04.2017 18:10:35</Time>
  <Exception>
    <Source>System</Source>
    <Message>Fehler beim Senden von Mail.</Message>
    <Stack>   bei System.Net.Mail.SmtpClient.Send(MailMessage message)
   bei CardGame.Web.HtmlHelpers.EmailHelper.SendEmail(String emailaddress, String emailsubject, String text) in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.Web\HtmlHelpers\EmailHelper.cs:Zeile 38.</Stack>
    <InnerException>
      <Source>System</Source>
      <Message>Der Remotename konnte nicht aufgelöst werden: 'srv08.itccn.loc'</Message>
      <Stack>   bei System.Net.ServicePoint.GetConnection(PooledStream PooledStream, Object owner, Boolean async, IPAddress&amp; address, Socket&amp; abortSocket, Socket&amp; abortSocket6)
   bei System.Net.PooledStream.Activate(Object owningObject, Boolean async, GeneralAsyncDelegate asyncCallback)
   bei System.Net.ConnectionPool.GetConnection(Object owningObject, GeneralAsyncDelegate asyncCallback, Int32 creationTimeout)
   bei System.Net.Mail.SmtpConnection.GetConnection(ServicePoint servicePoint)
   bei System.Net.Mail.SmtpClient.GetConnection()
   bei System.Net.Mail.SmtpClient.Send(MailMessage message)</Stack>
    </InnerException>
  </Exception>
</Xcpt>
<Info>
  <Time>27.04.2017 18:10:48</Time>
  <Info>id: 11</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:48</Time>
  <Info>numPacks: 1</Info>
</Info>
<Xcpt>
  <Time>27.04.2017 18:10:49</Time>
  <Exception>
    <Source>System</Source>
    <Message>Fehler beim Senden von Mail.</Message>
    <Stack>   bei System.Net.Mail.SmtpClient.Send(MailMessage message)
   bei CardGame.Web.HtmlHelpers.EmailHelper.SendEmail(String emailaddress, String emailsubject, String text) in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.Web\HtmlHelpers\EmailHelper.cs:Zeile 38.</Stack>
    <InnerException>
      <Source>System</Source>
      <Message>Der Remotename konnte nicht aufgelöst werden: 'srv08.itccn.loc'</Message>
      <Stack>   bei System.Net.ServicePoint.GetConnection(PooledStream PooledStream, Object owner, Boolean async, IPAddress&amp; address, Socket&amp; abortSocket, Socket&amp; abortSocket6)
   bei System.Net.PooledStream.Activate(Object owningObject, Boolean async, GeneralAsyncDelegate asyncCallback)
   bei System.Net.ConnectionPool.GetConnection(Object owningObject, GeneralAsyncDelegate asyncCallback, Int32 creationTimeout)
   bei System.Net.Mail.SmtpConnection.GetConnection(ServicePoint servicePoint)
   bei System.Net.Mail.SmtpClient.GetConnection()
   bei System.Net.Mail.SmtpClient.Send(MailMessage message)</Stack>
    </InnerException>
  </Exception>
</Xcpt>
<Info>
  <Time>27.04.2017 18:10:58</Time>
  <Info>id: 9</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:58</Time>
  <Info>numPacks: 1</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:59</Time>
  <Info>vID: 382</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:59</Time>
  <Info>Card: 319</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:59</Time>
  <Info>Card: 327</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:59</Time>
  <Info>Card: 239</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:59</Time>
  <Info>Card: 298</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:59</Time>
  <Info>Card: 37</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:59</Time>
  <Info>Card: 596</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:59</Time>
  <Info>Card: 612</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:59</Time>
  <Info>Card: 351</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:59</Time>
  <Info>Card: 72</Info>
</Info>
<Info>
  <Time>27.04.2017 18:10:59</Time>
  <Info>Card: 694</Info>
</Info>
<Xcpt>
  <Time>27.04.2017 18:10:59</Time>
  <Exception>
    <Source>System</Source>
    <Message>Fehler beim Senden von Mail.</Message>
    <Stack>   bei System.Net.Mail.SmtpClient.Send(MailMessage message)
   bei CardGame.Web.HtmlHelpers.EmailHelper.SendEmail(String emailaddress, String emailsubject, String text) in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.Web\HtmlHelpers\EmailHelper.cs:Zeile 38.</Stack>
    <InnerException>
      <Source>System</Source>
      <Message>Der Remotename konnte nicht aufgelöst werden: 'srv08.itccn.loc'</Message>
      <Stack>   bei System.Net.ServicePoint.GetConnection(PooledStream PooledStream, Object owner, Boolean async, IPAddress&amp; address, Socket&amp; abortSocket, Socket&amp; abortSocket6)
   bei System.Net.PooledStream.Activate(Object owningObject, Boolean async, GeneralAsyncDelegate asyncCallback)
   bei System.Net.ConnectionPool.GetConnection(Object owningObject, GeneralAsyncDelegate asyncCallback, Int32 creationTimeout)
   bei System.Net.Mail.SmtpConnection.GetConnection(ServicePoint servicePoint)
   bei System.Net.Mail.SmtpClient.GetConnection()
   bei System.Net.Mail.SmtpClient.Send(MailMessage message)</Stack>
    </InnerException>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 18:12:04</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Info>
  <Time>27.04.2017 19:25:19</Time>
  <Info>Entered Pass = stefan</Info>
</Info>
<Info>
  <Time>27.04.2017 19:25:19</Time>
  <Info>HashPass = D26F2BAD2D39019187121BE939D72729130D070AEFF8647926D9720F12AAA433C1B9B7783C462DDBC97EB8D198095021761C5E8242D434BBA2C146B51FE57BCC</Info>
</Info>
<Xcpt>
  <Time>27.04.2017 19:25:19</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 19:30:38</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 19:41:41</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 19:44:24</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 19:50:50</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 20:00:22</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 20:22:24</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 20:35:50</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 20:40:08</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 20:47:51</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 20:55:40</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 21:11:47</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 21:13:04</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 21:20:19</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 21:32:13</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 21:45:22</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 21:50:42</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 21:55:43</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 21:55:57</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 21:59:47</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 22:02:00</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 22:03:59</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 22:09:55</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 22:17:15</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 22:20:58</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 22:21:48</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 22:24:18</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Info>
  <Time>27.04.2017 22:28:20</Time>
  <Info>Entered Pass = stefan</Info>
</Info>
<Info>
  <Time>27.04.2017 22:28:20</Time>
  <Info>HashPass = D26F2BAD2D39019187121BE939D72729130D070AEFF8647926D9720F12AAA433C1B9B7783C462DDBC97EB8D198095021761C5E8242D434BBA2C146B51FE57BCC</Info>
</Info>
<Xcpt>
  <Time>27.04.2017 22:28:20</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 22:38:11</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 22:44:25</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 22:46:13</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 22:52:36</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 23:06:00</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 23:16:52</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 23:20:17</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 23:25:44</Time>
  <Exception>
    <Source>CardGame.Web</Source>
    <Message>Der Objektverweis wurde nicht auf eine Objektinstanz festgelegt.</Message>
    <Stack>   bei CardGame.Web.Controllers.CardPackController.PackOverview(Int32 page) in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.Web\Controllers\CardPackController.cs:Zeile 34.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 23:27:30</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 23:29:33</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 23:30:44</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 23:32:17</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Info>
  <Time>27.04.2017 23:32:54</Time>
  <Info>GetNumUsers 21</Info>
</Info>
<Info>
  <Time>27.04.2017 23:32:54</Time>
  <Info>GetNumCards 382</Info>
</Info>
<Info>
  <Time>27.04.2017 23:32:54</Time>
  <Info>GetNumDecks 7</Info>
</Info>
<Info>
  <Time>27.04.2017 23:34:46</Time>
  <Info>GetNumUsers 21</Info>
</Info>
<Info>
  <Time>27.04.2017 23:34:46</Time>
  <Info>GetNumCards 382</Info>
</Info>
<Info>
  <Time>27.04.2017 23:34:46</Time>
  <Info>GetNumDecks 7</Info>
</Info>
<Xcpt>
  <Time>27.04.2017 23:34:46</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Info>
  <Time>27.04.2017 23:38:01</Time>
  <Info>GetNumUsers 21</Info>
</Info>
<Info>
  <Time>27.04.2017 23:38:01</Time>
  <Info>GetNumCards 382</Info>
</Info>
<Info>
  <Time>27.04.2017 23:38:01</Time>
  <Info>GetNumDecks 7</Info>
</Info>
<Xcpt>
  <Time>27.04.2017 23:38:01</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Info>
  <Time>27.04.2017 23:38:36</Time>
  <Info>Entered Pass = stefan</Info>
</Info>
<Info>
  <Time>27.04.2017 23:38:36</Time>
  <Info>HashPass = D26F2BAD2D39019187121BE939D72729130D070AEFF8647926D9720F12AAA433C1B9B7783C462DDBC97EB8D198095021761C5E8242D434BBA2C146B51FE57BCC</Info>
</Info>
<Info>
  <Time>27.04.2017 23:48:53</Time>
  <Info>GetNumUsers 21</Info>
</Info>
<Info>
  <Time>27.04.2017 23:48:53</Time>
  <Info>GetNumCards 382</Info>
</Info>
<Xcpt>
  <Time>27.04.2017 23:48:54</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 23:48:54</Time>
  <Exception>
    <Source>CardGame.Web</Source>
    <Message>Der Objektverweis wurde nicht auf eine Objektinstanz festgelegt.</Message>
    <Stack>   bei CardGame.Web.Controllers.CardPackController.PackOverview(Int32 page) in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.Web\Controllers\CardPackController.cs:Zeile 34.</Stack>
  </Exception>
</Xcpt>
<Info>
  <Time>27.04.2017 23:48:56</Time>
  <Info>GetNumUsers 21</Info>
</Info>
<Info>
  <Time>27.04.2017 23:48:56</Time>
  <Info>GetNumCards 382</Info>
</Info>
<Info>
  <Time>27.04.2017 23:48:56</Time>
  <Info>GetNumDecks 7</Info>
</Info>
<Xcpt>
  <Time>27.04.2017 23:50:22</Time>
  <Exception>
    <Source>CardGame.Web</Source>
    <Message>Der Objektverweis wurde nicht auf eine Objektinstanz festgelegt.</Message>
    <Stack>   bei CardGame.Web.Controllers.CardPackController.PackOverview(Int32 page) in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.Web\Controllers\CardPackController.cs:Zeile 34.</Stack>
  </Exception>
</Xcpt>
<Info>
  <Time>27.04.2017 23:50:24</Time>
  <Info>GetNumUsers 21</Info>
</Info>
<Info>
  <Time>27.04.2017 23:50:24</Time>
  <Info>GetNumCards 382</Info>
</Info>
<Info>
  <Time>27.04.2017 23:50:24</Time>
  <Info>GetNumDecks 7</Info>
</Info>
<Xcpt>
  <Time>27.04.2017 23:50:24</Time>
  <Exception>
    <Source>CardGame.Web</Source>
    <Message>Der Objektverweis wurde nicht auf eine Objektinstanz festgelegt.</Message>
    <Stack>   bei CardGame.Web.Controllers.CardPackController.PackOverview(Int32 page) in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.Web\Controllers\CardPackController.cs:Zeile 34.</Stack>
  </Exception>
</Xcpt>
<Info>
  <Time>27.04.2017 23:50:25</Time>
  <Info>GetNumUsers 21</Info>
</Info>
<Info>
  <Time>27.04.2017 23:50:25</Time>
  <Info>GetNumCards 382</Info>
</Info>
<Info>
  <Time>27.04.2017 23:50:25</Time>
  <Info>GetNumDecks 7</Info>
</Info>
<Info>
  <Time>27.04.2017 23:50:25</Time>
  <Info>GetNumUsers 21</Info>
</Info>
<Info>
  <Time>27.04.2017 23:50:25</Time>
  <Info>GetNumCards 382</Info>
</Info>
<Info>
  <Time>27.04.2017 23:50:25</Time>
  <Info>GetNumDecks 7</Info>
</Info>
<Info>
  <Time>27.04.2017 23:50:25</Time>
  <Info>GetNumUsers 21</Info>
</Info>
<Info>
  <Time>27.04.2017 23:50:25</Time>
  <Info>GetNumCards 382</Info>
</Info>
<Info>
  <Time>27.04.2017 23:50:25</Time>
  <Info>GetNumDecks 7</Info>
</Info>
<Info>
  <Time>27.04.2017 23:50:25</Time>
  <Info>GetNumUsers 21</Info>
</Info>
<Info>
  <Time>27.04.2017 23:50:25</Time>
  <Info>GetNumCards 382</Info>
</Info>
<Info>
  <Time>27.04.2017 23:50:25</Time>
  <Info>GetNumDecks 7</Info>
</Info>
<Info>
  <Time>27.04.2017 23:50:27</Time>
  <Info>GetNumUsers 21</Info>
</Info>
<Info>
  <Time>27.04.2017 23:50:27</Time>
  <Info>GetNumCards 382</Info>
</Info>
<Info>
  <Time>27.04.2017 23:50:27</Time>
  <Info>GetNumDecks 7</Info>
</Info>
<Xcpt>
  <Time>27.04.2017 23:50:54</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 23:51:20</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>27.04.2017 23:55:47</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
